iOS/Flows/Conversation/Cells/ConversationMessagesCell.swift

```diff
@@ -58,7 +58,7 @@ class ConversationMessagesCell: UICollectionViewCell, ConversationUIStateSettabl
     /// A reference to the current task that scrolls to a specific message
     private var scrollToMessageTask: Task<Void, Never>?
     /// If true we should scroll to the last item in the collection in layout subviews.
-    private var scrollToLastItemOnLayout: Bool = false
+    private var scrollToFirstUnreadIfNeccessary: Bool = false
```

Comment:
Now we attempt to scroll to the first unread message when a conversation is loaded.